pkg/skaffold/config/options.go

```diff
@@ -35,6 +35,8 @@ type PortForwardOptions struct {
 type SkaffoldOptions struct {
 	ConfigurationFile  string
 	GlobalConfig       string
+	DefaultRepo        string
+	DefaultRepoSet     bool
```

Comment:
`Set` is awkward. `Overridden`?  Or would it be easier to use a magic value (`<<unset>>`).